media/css/firefox/fights-for-you.scss

```diff
@@ -0,0 +1,234 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+$font-path: '/media/fonts';
+$image-path: '/media/protocol/img';
+
+@import '../../protocol/css/includes/lib';
+@import '../../protocol/css/components/button';
+@import '../../protocol/css/components/call-out';
+@import '../../protocol/css/components/feature-card';
+@import '../../protocol/css/components/hero';
+
+@import '../libs/system-font.css';
+
+
+.fffy-c-hero,
+.fffy-c-features,
+.fffy-c-footer-callout,
+.mzp-t-firefox h1,
+.mzp-t-firefox h2,
+.mzp-t-firefox h3,
+.mzp-t-firefox h4 {
+    font-family: system-ui, sans-serif;
+}
+
+$fffy-purple : #2A0140;
+$fffy-coral: #FF4F5D;
+$fffy-pink: #FF2889;
+
+.fffy-c-hero {
+    background-color: $fffy-purple;
+    background-image: url('/media/img/firefox/fights-for-you/noodle.svg'), url('/media/img/firefox/fights-for-you/chevron.svg');
+    background-position: top 200px left -60px, top -80px right calc(30% - 220px);
```

Comment:
Yeaaaaaah, that would be bad.

Reply:
> Rather than risking it I added a line of CSS in conditional comments to just use a plain purple background for IE

This seems like a logical approach, I just hope the crash is dependent on the properties applied, and not when parsing the CSS. I guess we can try and monitor in GA and see if there are IE9 visits to the page. 

Maybe @craigcook may have an IE9 VM to help test in?

Reply:
I was testing in an IE9 VM 🤷‍♀️